react/__tests__/MiniCart.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { render } from 'test-utils'
+import { fireEvent } from 'react-testing-library'
+
+import MiniCart from './../index'
+
+describe('<MiniCart /> component', () => {
+  const renderComponent = customProps => {
+    const props = {
+      type: 'popup',
+      hideContent: false,
+      ...customProps,
+    }
+    return render(<MiniCart {...props} />)
+  }
+
+  it('should be rendered', () => {
+    expect(renderComponent().asFragment()).toBeDefined()
+  })
+
+  it('should match the snapshot in popup mode', () => {
+    const wrapper = renderComponent()
+    const leftClick = { button: 0 }
+    const { getByText } = wrapper
+    fireEvent.click(getByText('Button Test'), leftClick)
+    expect(wrapper.asFragment()).toMatchSnapshot()
```

Comment:
Some line breaks would be good.

I usually follow the Arrange Act Assert.

In this case would be:
```js
const leftClick = { button: 0 }

const { getByText } = render(<MiniCart type="popup" hideContent={false} />)
fireEvent.click(getByText('Button Test'), leftClick)

expect(wrapper.asFragment()).toMatchSnapshot()
```